### ISSUE TITLE
F.local_response_normalization is not exported to the ONNX format

### ISSUE THREAD

**Issue by oyamay** [chainer/onnx-chainer#21]
When a model contains LRNs (`F.local_response_normalization`), the LRN nodes are not exported to the ONNX format correctly.
Other normalization operations such as BN (`L.BatchNormalization`) work fine.

This is an example code:
```
#!/usr/bin/env python3

import numpy as np
import chainer
import chainer.links as L
import chainer.functions as F
import onnx_chainer

class Alex(chainer.Chain):
    insize = 227

    def __init__(self, bn=False):
        super(Alex, self).__init__()
        with self.init_scope():
            self.conv1 = L.Convolution2D(None,  96, 11, stride=4)
            self.conv2 = L.Convolution2D(None, 256,  5, pad=2)
            self.conv3 = L.Convolution2D(None, 384,  3, pad=1)
            self.conv4 = L.Convolution2D(None, 384,  3, pad=1)
            self.conv5 = L.Convolution2D(None, 256,  3, pad=1)
            self.fc6 = L.Linear(None, 4096)
            self.fc7 = L.Linear(None, 4096)
            self.fc8 = L.Linear(None, 1000)
            self.norm1 = L.BatchNormalization(96)  if bn else F.local_response_normalization
            self.norm2 = L.BatchNormalization(256) if bn else F.local_response_normalization

    def __call__(self, x):
        h = F.max_pooling_2d(self.norm1(F.relu(self.conv1(x))), 3, stride=2)
        h = F.max_pooling_2d(self.norm2(F.relu(self.conv2(h))), 3, stride=2)
        h = F.relu(self.conv3(h))
        h = F.relu(self.conv4(h))
        h = F.max_pooling_2d(F.relu(self.conv5(h)), 3, stride=2)
        h = F.dropout(F.relu(self.fc6(h)))
        h = F.dropout(F.relu(self.fc7(h)))
        h = self.fc8(h)
        return h

x = np.zeros((1, 3, Alex.insize, Alex.insize), dtype=np.float32)
chainer.config.train = False

model_onnx = onnx_chainer.export(Alex(), x, filename='alex.onnx')
for node in model_onnx.graph.node:
    print(node.op_type)

print()

model_bn_onnx = onnx_chainer.export(Alex(True), x, filename='alex_bn.onnx')
for node in model_bn_onnx.graph.node:
    print(node.op_type)
```

This is the output:
```
Conv
Relu
MaxPool
Conv
Relu
MaxPool
Conv
Relu
Conv
Relu
Conv
Relu
MaxPool
Reshape
FC
Relu
FC
Relu
FC

Conv
Relu
BatchNormalization
MaxPool
Conv
Relu
BatchNormalization
MaxPool
Conv
Relu
Conv
Relu
Conv
Relu
MaxPool
Reshape
FC
Relu
FC
Relu
FC
```

The software versions I'm using are as follows:
- Python: 3.6.4
   - chainer: 4.2.0
   - numpy: 1.14.2
   - cupy: 4.2.0
   - protobuf: 3.5.2.post1
   - onnx: 1.0.0
   - onnx-chainer: 1.0.0a1
- libprotoc: 3.3.2
- CUDA: 8.0.61

**Comment by oyamay** [comment]
It seems that the problem has been solved in onnx-chainer 1.1.1a2. Thanks!
